auto_landing/include/auto_landing/tracking.hpp

```diff
@@ -0,0 +1,36 @@
+#include<ros/ros.h>
+
+#include<geometry_msgs/Twist.h>
+#include<geometry_msgs/PoseStamped.h>
+
+#include<nav_msgs/Odometry.h>
+
+#include<std_srvs/Trigger.h>
+
+namespace ariitk::auto_landing {
+    
+class Tracking {
+    public:
+        // Tracking();
+        void init(ros::NodeHandle& nh);
+        void run();
+
+    // private:
+    
+        void poseClbk(const nav_msgs::Odometry& msg);
+        bool serverClbk(std_srvs::Trigger::Request &req, std_srvs::Trigger::Response &resp);
+    
+        int count=0;
+        int a_=2;
```

Comment:
What does a_ signify? Please give a more descriptive name to the variable. Also don't initialize variables here, use the constructor for that.